draft-ietf-jsonpath-base.md

```diff
@@ -834,11 +847,12 @@ Queries:
 
 The examples in {{tbl-name}} show the name selector in use by child segments:
 
-| Query | Result | Result Paths | Comment |
-| :---: | ------ | :----------: | ------- |
-| `$.o['j j']['k.k']`   | `3` | `$['o']['j j']['k.k']`      | Named value in nested object      |
-| `$.o["j j"]["k.k"]`   | `3` | `$['o']['j j']['k.k']`      | Named value in nested object      |
-| `$["'"]["@"]` | `2` | `$['\'']['@']` | Unusual member names
+| Query                | Result       | Result Paths            | Comment                      |
+| :---:                | ------       | :----------:            | -------                      |
+| `$.o['j j']`        | `{"k.k": 3}` | `$['o']['j j']`        | Named value in nested object |
```

Comment:
It's probably accurate, but having the same comment for this as for the following two rows doesn't read very well.

Reply:
Yes, I wondered about that.  "nested object" fits both a single and a dual level of nesting...

Reply:
See commit below...